app/helpers/volunteer_ops_helper.rb

```diff
@@ -2,4 +2,10 @@ module VolunteerOpsHelper
   def button_text object
     object.new_record? ? 'Create a Volunteer Opportunity' : 'Update a Volunteer Opportunity'
   end
+
+  def link_to_vol_op(obj, name, html_options = {})
+    html_options.merge!(target: '_blank') unless obj.source == 'local'
```

Comment:
@ZmagoD if you can fix the above I think we're ready to merge - can make the change myself at some point if you don't get to it - but trying to tidy up a few other things first ...